src/sass/components/_advantages.scss

```diff
@@ -1,3 +1,94 @@
 .advantages {
-  background-color: pink;
+  background-color: $pageBg;
+  padding: 39px 0 59px 0;
+  &__header-hidden {
+    display: none;
+  }
+  &__list-header {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    margin-bottom: 38px;
+    @include font-mixin('Titan One', 400, 36px, 1.14, 0.04em, $brand-red);
+    text-transform: uppercase;
+    &--1::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-1-33.png);
+    }
+    &--2::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-2-33.png);
+    }
+    &--3::before {
+      @extend %advantages-icon;
+      background-image: url(../images/icon-3-33.png);
+    }
+  }
+  &__list-text {
+    @include font-mixin('DM Sans', 700, 12px, 1.67, normal, $body);
+    text-transform: none;
+    max-width: 230px;
+  }
+}
+
+%advantages-icon {
+  content: '';
+  height: 50px;
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+@media screen and (min-width: 768px) {
+  .advantages {
+    min-width: 768px;
+    flex-direction: row;
+    padding: 66px 0 140px 0;
+    &__list {
+      display: flex;
+    }
+    &__list-header {
+      font-size: 48px;
+      text-transform: none;
+      max-width: 180px;
+      margin-right: 60px;
+    }
+    &__list-text {
+      font-size: 14px;
+      line-height: 1.71;
+    }
+  }
+}
+
+@media screen and (min-width: 1280px) {
```

Comment:
deskto jest na 1200px :)

Reply:
Nie zgodze się. 1280px to standardowy punkt przełamania wykorzystywany np. w Tailwindzie.  

"Responsywny układ, punkty przerwania 480px, 768px, 1280px. W przypadku mobilnej wersji witryny układ jest responsywny. Witryna musi być poprawnie wyświetlana we wszystkich możliwych rozmiarach okna przeglądarki i na wszystkich urządzeniach, począwszy od szerokości 320px, zarówno w orientacji pionowej, jak i poziomej.
"

Reply:
o jezu nie zauwarzyłem w wytycznych, jest 1280px xd